tsconfig.json

```diff
@@ -7,9 +7,13 @@
     "typeRoots": ["./node_modules/@types", "./types"],
     "paths": {
       "@trpc/server": ["packages/server/src"],
-      "@trpc/react-query": ["packages/react-query/src"],
+      "@trpc/server/*": ["packages/server/src/*"],
```

Comment:
Required to cut src from the imports because we import from `@trpc/server/adapters/fetch`